mathematica/integrator_plots.cpp

```diff
@@ -219,137 +328,337 @@ class WorkErrorGraphGenerator {
     }
     CHECK_OK(bundle.Join());
 
-    std::vector<std::string> q_error_data;
-    std::vector<std::string> v_error_data;
-    std::vector<std::string> e_error_data;
+    std::vector<std::vector<std::vector<std::tuple<double, Length>>>>
```

Comment:
`std::tuple` in this case is just `std::pair`.  But in general I'd prefer to invest in a struct to give names to the components.  Also, naming the vectors would be good because it's really hard to figure out the purpose of each level.

Reply:
This is pretty much directly what we put in the Mathematica output, so we end up with nasty lists of lists of lists of lists anyway.
And we do not have support for pair in ToMathematica.